### PR TITLE
Add three featured data articles and five news articles

### PR DESCRIPTION
**Pull request by kzollove**
This PR adds Featured Data articles from July, September, and November 2021

### CODE REVIEW

File: templates/data/featured-jul-2021.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# Featured Data
+
+## Effects of nutrient enrichment on grassland biomass and plant diversity across the globe
+
+July 2021
+
+### Featured Data Packages
```
Comment (clnsmth) [inline templates/data/featured-jul-2021.md]: Should be "Citation"

File: templates/data/featured-jul-2021.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+1. Borer, E.T., W.S. Harpole, P.B. Adler et al. 2020. Data from nutrients cause grassland biomass to outpace herbivory, 2008-2019 ver 1. Environmental Data Initiative. [https://doi.org/10.6073/pasta/a318fe0fb11eb43c1a2c8233b2e3494f](https://doi.org/10.6073/pasta/a318fe0fb11eb43c1a2c8233b2e3494f).
```
Comment (clnsmth) [inline templates/data/featured-jul-2021.md]: Space separate the citations, not enumerated list

File: templates/data/featured-jul-2021.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+1. Seabloom, E.W., P.B. Adler, J. Alberti et al. 2020. Effects of chronic nutrient enrichment on plant diversity and ecosystem productivity, 2008-2019 ver 1. Environmental Data Initiative. [https://doi.org/10.6073/pasta/d78e6e48df246a03129c85a6c1da00d6](https://doi.org/10.6073/pasta/d78e6e48df246a03129c85a6c1da00d6).
+
```
Comment (clnsmth) [inline templates/data/featured-jul-2021.md]: Missing section header "Description"

File: templates/data/featured-nov-2021.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
+# Featured Data
+
+## Journey North – Hummingbird and Monarch Butterfly observations by volunteer community scientists across Central and North America (1996-2020)
+
+November 2021
+
+### Citation
+
+Sheehan, N. and L. Weber-Grullon. 2021. Journey North – Monarch Butterfly and Milkweed observations by volunteer community scientists across Central and North America (1996-2020) ver 1. Environmental Data Initiative. [https://doi.org/10.6073/pasta/f7d7bef57f94b33b8a18a26954252412](https://doi.org/10.6073/pasta/f7d7bef57f94b33b8a18a26954252412).
+Sheehan, N. and L. Weber-Grullon. 2021. Journey North – Hummingbird observations by volunteer community scientists across Central and North America (1996-2020) ver 1. Environmental Data Initiative. [https://doi.org/10.6073/pasta/a4bebd2b5fb5c1fd0757eac254a06357](https://doi.org/10.6073/pasta/a4bebd2b5fb5c1fd0757eac254a06357).
```
Comment (clnsmth) [inline templates/data/featured-nov-2021.md]: Should be new line separated.

File: templates/data/featured-sep-2021.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
+# Featured Data
+
+## Calling activity of Birds in the White Mountain National Forest: Audio Recordings (2016 and 2018)
+
+September 2021
+
+### Citation
+
+Symes, L.B., K.D. Kittelberger, S.M. Stone, R.T. Holmes, J.S. Ralston, I.P. Casteneda Ruvalcaba, M.S. Webster, and M.P. Ayres. 2021. Calling activity of Birds in the White Mountain National Forest: Audio Recordings (2016 and 2018) ver 3. Environmental Data Initiative. [https://doi.org/10.6073/pasta/efba421dc87e077ee28010069dac7c0d](https://doi.org/10.6073/pasta/efba421dc87e077ee28010069dac7c0d)
+
```
Comment (clnsmth) [inline templates/data/featured-sep-2021.md]: Missing "Description" header

File: templates/data/news-211102a.md [hunk 6]
```diff
@@ -0,0 +1,9 @@
+# news-tags
```
Comment (clnsmth) [inline templates/data/news-211102a.md]: Thanks for including these. Please move to the bottom of the page and wrap in HTML comment tags `<!-- a normal html comment -->`

File: templates/data/news-211102a.md [hunk 7]
```diff
@@ -0,0 +1,9 @@
...
+News
+
+# date
```
Comment (clnsmth) [inline templates/data/news-211102a.md]: Use the same date format as featured data

File: templates/data/news-211102a.md (hunk 6 above)
Comment (clnsmth) [reply templates/data/news-211102a.md]: Use the title "News" on line 1 as done on featured data pages.

File: templates/data/news-211102a.md [hunk 8]
```diff
@@ -0,0 +1,9 @@
...
+2021-11-02
+
+# Updates to user-contributed journal citation interface on the EDI Data Portal
```
Comment (clnsmth) [inline templates/data/news-211102a.md]: Second title on the page should be the article title. Again similar to featured data

File: templates/data/news-211102b.md [hunk 9]
```diff
@@ -0,0 +1,101 @@
+# news-tags
```
Comment (clnsmth) [inline templates/data/news-211102b.md]: Same comments of news-211202*.md apply to this and subsequent files

**Review by clnsmth — CHANGES_REQUESTED** [review]
Thanks @kzollove. A couple requested changes.

- Try getting the image file size below 200 KB. Maybe compress again?
- Apply file name format of featured data to news items
- Some formatting tweaks, noted in the files

**Comment by kzollove** [comment]
Formatting tweaks have been implemented across the board.

Images: It doesn't look like I can get the PNG size down further with tinypng. I can compress more if I use JPEG. Should I replace PNGs with JPEGs where they are larger than 200 KB?

Name format: How should I go about this (there are multiple news articles from the same month, some even from the same day) Possible: news-month-yyyy.md or news-day-month-yyyy, how to indicate different within month or day? news-month-yyyy-a.md, news-month-yyyy-1.md, etc?

**Comment by clnsmth** [comment]
Thanks @kzollove. 
* .jpgs work and if for some reason that doesn't do the trick, try resizing. 
* For dates across all articles (news, blog, featured) let's use `[articletype]-YYYYMMDDhhmm.md`. Reason being (just remembering this now) is that article management (storage and posting) will be determined by publication date. If there isn't a hhmm, fabricate one. : )

**Comment by kzollove** [comment]
@clnsmth thanks for the feedback.

All formatting tweaks implemented.
All images in the featured and news articles are <200 KB
All filenames are now formatted as `[articletype]-YYYYMMDDhhmm.md`

**Pull request merged**